### PR TITLE
Docs: Added Create React App + Typescript info

### PR DESCRIPTION
**Pull request by thomastvedt**
my first pull request @sapegin :) 

added missing info when using Create React App together with TypeScript. Had to read webpack-config from react-scripts-ts/config/webpack.config.dev.js.

### CODE REVIEW

File: docs/Webpack.md [hunk 1]
```diff
@@ -77,6 +77,16 @@ module.exports = {
 
 [Create React App](https://github.com/facebookincubator/create-react-app) is supported out of the box, you don’t even need to create a style guide config if your components could be found using a default glob pattern, `src/components/**/*.{js,jsx}`.
 
+If you're using Create react App together with Typescript, you have to install [react-docgen-typescript](https://github.com/styleguidist/react-docgen-typescript) and add a `components`, `webpackConfig` and `propsParser` section to your  `styleguide.config.js`:
+
+```javascript
+module.exports = {
+  components: 'src/components/**/*.{ts,tsx}'
```
Comment (sapegin) [inline docs/Webpack.md]: Missed comma here ;-|

File: docs/Webpack.md [hunk 2]
```diff
@@ -77,6 +77,16 @@ module.exports = {
...
+  propsParser: require('react-docgen-typescript').parse,
+  webpackConfig: require('./node_modules/react-scripts-ts/config/webpack.config.dev.js')
```
Comment (sapegin) [inline docs/Webpack.md]: Will `react-scripts-ts/config/webpack.config.dev.js` work too?

File: docs/Webpack.md [hunk 3]
```diff
@@ -77,6 +77,16 @@ module.exports = {
 
 [Create React App](https://github.com/facebookincubator/create-react-app) is supported out of the box, you don’t even need to create a style guide config if your components could be found using a default glob pattern, `src/components/**/*.{js,jsx}`.
 
+If you're using Create react App together with Typescript, you have to install [react-docgen-typescript](https://github.com/styleguidist/react-docgen-typescript) and add a `components`, `webpackConfig` and `propsParser` section to your  `styleguide.config.js`:
```
Comment (sapegin) [inline docs/Webpack.md]: have → need.

Create React App, TypeScript — for consistency.

add a `components`, `webpackConfig` and `propsParser` section to your  `styleguide.config.js` → create a  `styleguide.config.js` file (most likely user won’t have any config at this time).

File: docs/Webpack.md (hunk 1 above)
Comment (thomastvedt) [reply docs/Webpack.md]: fixed :)

**Comment by sapegin** [comment]
Thanks, merged! 

💥  And congrats with your first PR, I'm super to have it in Styleguidist! 💥

**Pull request merged**